src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+  }
```

Comment:
The constructor of the Animal class should add the newly created animal instance to the static alive array. This logic is currently duplicated in both Herbivore and Carnivore constructors which is not efficient.